test/system-test/test_constitution.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import tempfile
+
+from utils import apply_kms_constitution, propose, get_test_action, get_test_proposal
+
+def test_constitution_auto_accept(setup_kms):
+    with get_test_action() as action_file:
+        apply_kms_constitution(actions=action_file.name)
```

Comment:
do you have multiple nodes voting in this test?

Reply:
We don't have any voting in this test, correct me if I'm wrong but there's no standalone scripts for voting in the KMS, it's all baked into scripts doing other things mainly proposing.

I add scripts in the next PR since we will want tests for it, but in this PR the change is just about setting constitutions and showing that a proposal is open is enough to demonstrate whether the resolve function auto accepts or not